charts/trino/templates/tests/test-jmx.yaml

```diff
@@ -1,4 +1,8 @@
-{{- if or .Values.jmx.exporter.enabled .Values.serviceMonitor.enabled }}
+{{- $coordinatorJmx := merge .Values.jmx.coordinator (omit .Values.jmx "coordinator" "worker") -}}
```

Comment:
Is it possible to define these merged values in `_helpers.tpl`?

Reply:
Problem with that is that you would need to declare them as "defines" which transforms them in strings. so you cannot then do stuff like `{{- if $coordinatorJmx.enabled -}}`. 
I struggled a lot with this, and I agree that it is not the most elegant thing to have these variables declared multiple times, but it really was the least of evils. 
Maybe there is a better way of doing this, but this is the best I could come up with.

Reply:
Also fun fact: if you omit the `omit` clause, Helm throws stack overflow! It has been a long time since I last got one of those...

Reply:
It's elegant enough :D